model/src/channel/message/flags.rs

```diff
@@ -11,6 +11,7 @@ bitflags! {
         const SUPPRESS_EMBEDS = 1 << 2;
         const SOURCE_MESSAGE_DELETED = 1 << 3;
         const URGENT = 1 << 4;
+        const HAS_THREAD = 1 << 5;
```

Comment:
Can we document what it means for a message to have a thread? It's not very clear without thinking about how this is displayed in a UI.